gravitee-apim-portal-webui/src/app/components/gv-alert/gv-alert.component.ts

```diff
@@ -72,6 +74,7 @@ export class GvAlertComponent implements OnInit {
   apisOptions: { label: string; value: string }[];
   apiAllLabel: string;
   isWebhookNotifierEnabled = false;
+  private unsubscribe$: Subject<boolean> = new Subject<boolean>();
```

Comment:
juste for info i prefer use with void and not with boolean `private unsubscribe$: new Subject<void>;`  and then just `this.unsubscribe$.next();` because true or false is not important we just need an event. 
but it's only style

Reply:
yes, I applied your suggestion, thanks